tests/core/test_external_search.py

```diff
@@ -5092,18 +5092,30 @@ def _leaf(key, value, op="eq"):
     def _jq(query):
         return JSONQuery(dict(query=query))
 
+    match_args = dict(
+        auto_generate_synonyms_phrase_query=False,
+        max_expansions=10,
+        fuzziness="AUTO",
+    )
```

Comment:
Might be worth replacing the arguments to this with constants that are defined somewhere and also used at the same call in [core/external_search.py](https://github.com/ThePalaceProject/circulation/pull/702/files#diff-bd35fcc1f1282a6accc97a70b10cf52835dc43297097ea923569bfd4d0a1b13f).

You know someone will update the parameters in one place and forget the other, some day. :slightly_smiling_face:

Reply:
Sounds about right, both places now use the same source.